pyproject.toml

```diff
@@ -52,7 +52,26 @@ target-version = "py39"
 quote-style = "preserve"
 
 [tool.ruff.lint]
-ignore = ["E731", "E741"]
+select = [
+    "A",  # flake8-builtins
+    "E",  # pycodestyle
+    "ERA",  # eradicate
+    "F",  # pyflakes
+    "I",  # isort
+    "ICN",  # flake8-import-conventions
+    "ISC",  # flake8-implicit-str-concat
+    "N",  # pep8-naming
+    "PIE",  # flake8-pie
+    "T20",  # flake8-(p)print
+    "UP",  # pyupgrade
+]
```

Comment:
The default configuration is posted here: https://docs.astral.sh/ruff/configuration/ . As you can see, the default selected set of rules is `select = ["E4", "E7", "E9", "F"]`, which is not enough in my opinion.